src/antq/util/maven.clj

```diff
@@ -79,11 +79,24 @@
      :artifact artifact
      :remote-repos remote-repos}))
 
-(defn ^Model read-pom
+(defn- ^Model read-pom*
   [^String url]
   (with-open [reader (io/reader url)]
     (.read (MavenXpp3Reader.) reader)))
 
+(defn ^Model read-pom
+  [^String url]
+  (loop []
+    (or (try
+          (read-pom* url)
+          (catch java.net.ConnectException e
+            (if (= "Operation timed out" (ex-message e))
+              (log/error (str "Fetching pom from " url " failed because it timed out, retrying"))
+              (throw e))))
+        (recur))))
```

Comment:
Like src/antq/util/git.clj, could you set an upper limit on the number of retries?